modules/sievefilters/modules.php

```diff
@@ -1160,108 +1166,102 @@ public function process()
+        $res .= get_blocked_senders($mailbox, $mailbox['id'], 'x-circle-fill', 'globe-europe-africa', $this->get('site_config'), $this->get('user_config'), $this);
+        $res .= '</tbody></table>';
+        $res .= '</div></div></div>';
+        $this->out('sieve_detail_display', $res);
     }
 }
 
 /**
  * @subpackage sievefilters/output
  */
-class Hm_Output_sievefilters_settings_accounts extends Hm_Output_Module {
+class Hm_Output_account_sieve_filters extends Hm_Output_Module {
     protected function output() {
-        if (!$this->get('sieve_filters_enabled')) {
-            return '<div class="empty_list">' . $this->trans('Sieve filter is deactivated') . '</div>';
+        if (! ($mailbox = $this->get('mailbox')) || empty($mailbox['sieve_config_host']) || !$this->get('sieve_filters_enabled')) {
+            return;
         }
-        $mailboxes = $this->get('imap_accounts', array());
-        $res = get_classic_filter_modal_content();
-        $res .= get_script_modal_content();
-        $sieve_supported = 0;
-        foreach($mailboxes as $mailbox) {
-            if (empty($mailbox['sieve_config_host'])) {
-                continue;
-            }
-            $sieve_supported++;
-            $result = get_mailbox_filters($mailbox, $this->get('site_config'), $this->get('user_config'));
-            $num_filters = $result['count'];
-            $res .= '<div class="sievefilters_accounts_item">';
-            $res .= '<div class="sievefilters_accounts_title settings_subtitle py-2 d-flex justify-content-between border-bottom cursor-pointer">' . $mailbox['name'];
-            $res .= '<span class="filters_count">' . sprintf($this->trans('%s filters'), $num_filters) . '</span></div>';
-            $res .= '<div class="sievefilters_accounts filter_block p-3 d-none"><div class="filter_subblock">';
-            $res .= '<button class="add_filter btn btn-primary" account="'.$mailbox['name'].'">Add Filter</button> <button  account="'.$mailbox['name'].'" class="add_script btn btn-light border">Add Script</button>';
-            $res .= '<table class="filter_details table my-3"><tbody>';
-            $res .= '<tr><th class="text-secondary fw-light col-sm-1">Priority</th><th class="text-secondary fw-light col-sm-9">Name</th><th class="text-secondary fw-light col-sm-2">Actions</th></tr>';
-            $res .= $result['list'];
-            $res .= '</tbody></table>';
-            $res .= '<div class="mb-3 d-none">
-                            <div class="d-block">
-                                <h3 class="mb-1">If conditions are not met</h3>
-                                <small>Define the actions if conditions are not met. If no actions are provided the next filter will be executed. If there are no other filters to be executed, the email will be delivered as expected.</small>
-                            </div>
+        if (session_status() === PHP_SESSION_ACTIVE) {
```

Comment:
Cypht uses custom session class to handle any interaction with the session. It includes close_early method. Please check how it is used in other places and call accoringly.
Did you check if anything is written in the session after this place of code?

Reply:
I checked, but I didn't observe any changes to the session

Reply:
Sorry I forgot to remove this line. I have used the cypht session  `close_early` method in the `load_account_sieve_filters` handler